src/util/nice-time.test.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-env jest */
+
+import niceTime from 'src/util/nice-time'
+
+describe('niceTime tests', () => {
+    test('should return now for timeperiod of less than 90 seconds', () => {
+        const date = new Date()
+        expect(niceTime(date)).toBe('now')
+    })
+
+    test('should return minutes for timeperiod of less than 600 seconds', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setMinutes(date.getMinutes() + 20)
+        expect(niceTime(date, {now})).toBe('20 minutes ago')
+    })
+
+    test('should return timeperiod stamp for timeperiod less than 24 hours', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setHours(date.getHours() + 5)
+        expect(niceTime(date, {now})).toBe(`${('0' + date.getHours()).slice(-2)}:${('0' + date.getMinutes()).slice(-2)}`)
+    })
+
+    test('should return the timeperiod stamp and the day for timeperiod less than 24 hours but not on the same day', () => {
+        const date = new Date()
+        const now = new Date()
+        now.setHours(date.getHours() + 22)
+        expect(niceTime(date, {now})).toBe(`Yesterday ${('0' + date.getHours()).slice(-2)}:${('0' + date.getMinutes()).slice(-2)}`)
```

Comment:
The thing with doing ```now = new Date(2016, 7, 2, 14, 55)``` is, it would only show the string ``'Yesterday 14:55'``` if we pass the ```now``` date to it as well. I am really sorry, but I can't see a reason to change the test? I will change it irrespective, but I really would like to understand more clearly. Also the yesterday case is already tested and all of the other cases it can have.